src/app/fills/page.tsx

```diff
@@ -27,42 +27,19 @@ import IconButton from "@mui/material/IconButton";
 import FillTableRow from "@/components/FillTableRow";
 
 import Stack from "@mui/material/Stack";
+import { useAppDispatch, useAppSelector } from "@/redux/hooks";
+import { addNewFill, removeFill } from "@/redux/fills/fillsSlice";
 
 const gotCompressor = dayjs("2025-01-01T00:00:00.000");
 
-function newFill(id) {
-  return {
-    id,
-    type: "air",
-    start: 0,
-    end: 3000,
-    o2: 20.9,
-    he: 0,
-    cylinder: "",
-  };
-}
-
 export default function About() {
-  const [error, setError] = React.useState(null);
-
-  const [personValue, setPerson] = React.useState(null);
-
-  const [typeValue, setType] = React.useState("air");
-
-  const [oxygenAmount, setOxygen] = React.useState("20.9");
-  const [heliumAmount, setHelium] = React.useState("0");
-  const [startPressure, setStartPressure] = React.useState("0");
-  const [endPressure, setEndPressure] = React.useState("0");
-
-  const [oxygenError, setOxygenError] = React.useState("");
-  const [heliumError, setHeliumError] = React.useState("");
-
-  const [startError, setStartError] = React.useState("");
-  const [endError, setEndError] = React.useState("");
```

Comment:
Moved this into redux. Will be easier to load and manage data from the backed with it stored in redux. Makes files easier to work in as well.